### PR TITLE
BZ-1711689: Updated PV access modes.

### PR DESCRIPTION
**Pull request by huffmanca**
Updated the access mode table to comment out any storage options that aren't included as of 4.1.

This is for OS 4.x.

### CODE REVIEW

File: modules/storage-persistent-storage-pv.adoc [hunk 1]
```diff
@@ -128,16 +128,17 @@ the Pods that use these volumes are deleted.
 |AWS EBS  | ✅ | - |  -
 |Azure File | ✅ | ✅ | ✅
 |Azure Disk | ✅ | - | -
```
Comment (liangxia) [inline modules/storage-persistent-storage-pv.adoc]: Azure (both file and disk) are not supported in OCP 4.1

Comment (huffmanca) [reply modules/storage-persistent-storage-pv.adoc]: Both of these lines have been commented out.

**Comment by huffmanca** [comment]
@liangxia ,

The Azure lines have been commented out; this should be ready for rereview.

**Comment by liangxia** [comment]
LGTM ( from QE )

**Pull request merged**